### PR TITLE
feat: add so limit can be provided to endpoint

### PR DESCRIPTION
**Pull request by davidsoderberg**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Make limit a query param
- **Why this change was needed?** (You can also link to an open issue here)
So larger amount of messages can be fetched with one request

### CODE REVIEW

File: apps/api/src/app/messages/usecases/get-messages/get-messages.usecase.ts [hunk 1]
```diff
@@ -7,7 +7,11 @@ export class GetMessages {
   constructor(private messageRepository: MessageRepository, private subscriberRepository: SubscriberRepository) {}
 
   async execute(command: GetMessagesCommand) {
-    const LIMIT = 10;
+    const LIMIT = command.limit;
+
+    if (LIMIT > 1000) {
```
Comment (davidsoderberg) [inline apps/api/src/app/messages/usecases/get-messages/get-messages.usecase.ts:12]: Is this validation needed? Should it be larger or smaller?

Comment (ainouzgali) [reply apps/api/src/app/messages/usecases/get-messages/get-messages.usecase.ts:12]: I am not sure. I tend to think it should be smaller. At least until a need for more comes up

**Review by ainouzgali — APPROVED** [review]
🤗

File: apps/api/src/app/messages/usecases/get-messages/get-messages.usecase.ts (hunk 1 above)
Comment (p-fernandez) [inline apps/api/src/app/messages/usecases/get-messages/get-messages.usecase.ts:12]: Same here. I would store the default upper limit in a constant to not have to hardcode its value everywhere and also to prepare it to be configurable in the future.
Also nit lowercase the variable as it is not a constant in this method.

**Review by p-fernandez — APPROVED** [review]

File: apps/api/src/app/messages/messages.controller.ts [hunk 2]
```diff
@@ -50,6 +57,7 @@ export class MessagesController {
         page: page ? Number(page) : 0,
         channel,
         subscriberId,
+        limit: limit ? Number(limit) : 10,
```
[the remark this answers is not in the document]
Comment (davidsoderberg) [reply apps/api/src/app/messages/messages.controller.ts:60]: I think both of your comments makes perfect sense and this is a perfect technical debt that we should apply everywhere, I would like to discuss both what hour default limit should be and what the upper bound should be 👍

**Pull request merged**